prow/config.yaml

```diff
@@ -5848,6 +5848,36 @@ postsubmits:
         - "--upload=gs://kubernetes-jenkins/logs"
 
 periodics:
+- interval: 15m
+  agent: kubernetes
+  name: application-periodic-default-gke
+  labels:
+    preset-service-account: true
+    preset-k8s-ssh: true
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20180418-79a97ca02-master
+      args:
+      - "--job=$(JOB_NAME)"
+      - "--repo=github.com/foxish/application=master"
```

Comment:
your own fork for now? xD

Reply:
Yeah, will change it once the testing is stable. Thanks for the review!